src/library/librarycontrol.h

```diff
@@ -67,29 +81,53 @@ class LibraryControl : public QObject {
   private:
     Library* m_pLibrary;
 
-    ControlObject* m_pSelectNextTrack;
-    ControlObject* m_pSelectPrevTrack;
-    ControlObject* m_pSelectTrack;
-
-    ControlObject* m_pSelectSidebarItem;
-    ControlObject* m_pSelectPrevSidebarItem;
-    ControlObject* m_pSelectNextSidebarItem;
-
-    ControlObject* m_pToggleSidebarItem;
-    ControlObject* m_pLoadSelectedIntoFirstStopped;
-    ControlObject* m_pAutoDjAddTop;
-    ControlObject* m_pAutoDjAddBottom;
-
-    ControlObject* m_pFontSizeKnob;
-    ControlPushButton* m_pFontSizeIncrement;
-    ControlPushButton* m_pFontSizeDecrement;
-
+    // Simulate pressing a key on the keyboard
+    void emitKeyEvent(QKeyEvent&& event);
+
+    // Controls to navigate vertically within currently focussed widget (up/down buttons)
+    std::unique_ptr<ControlPushButton> m_pMoveUp;
+    std::unique_ptr<ControlPushButton> m_pMoveDown;
+    std::unique_ptr<ControlObject> m_pMoveVertical;
+
+    // Controls to navigate horizontally within currently selected item (left/right buttons)
+    std::unique_ptr<ControlPushButton> m_pMoveLeft;
+    std::unique_ptr<ControlPushButton> m_pMoveRight;
+    std::unique_ptr<ControlObject> m_pMoveHorizontal;
+
+    // Controls to navigate between widgets (tab/shit+tab button)
+    std::unique_ptr<ControlPushButton> m_pMoveFocusForward;
+    std::unique_ptr<ControlPushButton> m_pMoveFocusBackward;
+    std::unique_ptr<ControlObject> m_pMoveFocus;
+
+    // Control to choose the currently selected item in focussed widget (double click)
```

Comment:
hah wow I've never seen that. ok

(Mixxx is written in American English anyway. We just did a big s/analyse/analyze/  :P)